tests/admin_views/tests.py

```diff
@@ -3300,7 +3300,7 @@ def test_custom_function_action_with_redirect(self):
 
     def test_default_redirect(self):
         """
-        Test that actions which don't return an HttpResponse are redirected to
+        Actions which don't return an HttpResponse are redirected to
```

Comment:
It would be nice to reflow the docstrings in cases like this (wrapping at 79 characters) so that the lines aren't unbalanced.